view/desktop/bin/Cargo.toml

```diff
@@ -67,3 +67,6 @@ mimalloc = { version = "0.1.39", default-features = false }
 objc = "0.2.7"
 cocoa = "0.26.0"
 macos-trampoline = { path = "./lib/macos" }
+
+[target.'cfg(not(any(target_os = "android", target_os = "ios")))'.dependencies]
```

Comment:
Do we really need this OS-specific clarification?

Reply:
I believe this is auto-generated by tauri. Although I suppose it's safe to delete them.